js/signErrorStyle.js

```diff
@@ -0,0 +1,61 @@
+import { $ } from './common.js';
+
+const $jsEmailErrorMsg = $('.js-email-error-msg');
```

Comment:
js는 selector에서만 사용하고 변수명에는 들어갈 필요가 없을 것 같아요!
```suggestion
const $emailErrorMsg = $('.js-email-error-msg');
```